src/Features/CSharp/Portable/CodeRefactorings/ConvertToInterpolatedString/ConvertToInterpolatedStringRefactoringProvider.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Composition;
+using System.Linq;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Simplification;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.ConvertToInterpolatedString
+{
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = PredefinedCodeRefactoringProviderNames.ConvertToInterpolatedString), Shared]
+    internal partial class ConvertToInterpolatedStringRefactoringProvider :
+        AbstractConvertToInterpolatedStringRefactoringProvider
+            <InterpolatedStringExpressionSyntax,
+             InterpolatedStringContentSyntax,
+             InterpolationSyntax,
+             InvocationExpressionSyntax,
+             ExpressionSyntax,
+             ArgumentSyntax,
+             LiteralExpressionSyntax>
+    {
+        protected override SeparatedSyntaxList<ArgumentSyntax>? GetArguments(InvocationExpressionSyntax invocation) =>
+            invocation?.ArgumentList?.Arguments;
+
+        protected override LiteralExpressionSyntax GetFirstArgument(SeparatedSyntaxList<ArgumentSyntax> arguments) =>
```

Comment:
You could remove this method entirely and replace it with:

```c#
syntaxFacts.GetExpressionOfArgument(arguments[0]) as TLiteralExpressionSyntax
```

in the shared code.